sdk/android/app/build.gradle

```diff
@@ -10,6 +10,9 @@ android {
         targetSdkVersion 23
         versionCode 1
         versionName "1.0"
+        ndk {
```

Comment:
can we add a comment to this? specifying that this is only because react-native does not have x64